### PR TITLE
JI:768968 P2022-1654 Fix translations in existed components

### PR DESCRIPTION
**Pull request by aniasgit**

### CODE REVIEW

**Review by krysia1 — APPROVED** [review]

File: libs/translations/src/feedback/en.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   "breadcrumb": {
     "breadcrumb": "breadcrumb",
     "save": "Save"
   },
+  "description": "Description",
+  "missingTitleError": "Title cannot be empty",
+  "newPresentation": "New presentation",
   "shareDialogMessage": "Congratulations, your presentation: {{name}} just started, use the link below to share it!",
   "sharePresentation": "Share presentation",
   "search": {
```
Comment (piotrrawski) [inline libs/translations/src/feedback/en.json:11]: if we want have translations in alphabetical order search shouldn't be before shares

File: libs/translations/src/feedback/pl.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   "breadcrumb": {
     "breadcrumb": "nawigacja",
     "save": "Zapisz"
   },
+  "description": "Opis",
+  "missingTitleError": "Tytuł nie może być pusty",
+  "newPresentation": "Nowa prezentacja",
   "shareDialogMessage": "Gratulacje, Twoja prezentacja: {{name}} właśnie się rozpoczęła, użyj poniższego linku, aby ją udostępnić!",
   "sharePresentation": "Udostępnij prezentację",
   "search": {
```
Comment (piotrrawski) [inline libs/translations/src/feedback/pl.json:11]: https://github.com/intive/patronage22-web-wroclaw/pull/62/files#r825753490

File: libs/translations/src/common/en.json [hunk 3]
```diff
@@ -6,14 +6,14 @@
...
-    "name": "Intive",
-    "wroclaw": "Wroclaw",
     "address": "Pilsudskiego 69, 50-019 Wroclaw",
-    "phone": "+48 71 769 59 00"
+    "name": "Intive",
+    "phone": "+48 71 769 59 00",
+    "wroclaw": "Wroclaw"
   },
   "linkCopied": "Link copied",
   "linkCopyingFailed": "Link copying failed!",
```
Comment (piotrrawski) [inline libs/translations/src/common/en.json:19]: in line 27,28 maxCharLength shouldn't be before minCharLength

File: libs/translations/src/common/pl.json [hunk 4]
```diff
@@ -6,14 +6,14 @@
...
-    "name": "Intive",
-    "wroclaw": "Wrocław",
     "address": "Piłsudskiego 69, 50-019 Wrocław",
-    "phone": "+48 71 769 59 00"
+    "name": "Intive",
+    "phone": "+48 71 769 59 00",
+    "wroclaw": "Wrocław"
   },
   "linkCopied": "Skopiowano link",
   "linkCopyingFailed": "Kopiowanie linku nie powiodło się!",
```
Comment (piotrrawski) [inline libs/translations/src/common/pl.json:19]: https://github.com/intive/patronage22-web-wroclaw/pull/62/files#r825759344

File: libs/translations/src/feedback/en.json (hunk 1 above)
Comment (aniasgit) [reply libs/translations/src/feedback/en.json:11]: right :)

File: libs/translations/src/common/en.json (hunk 3 above)
Comment (krysia1) [reply libs/translations/src/common/en.json:19]: Oh and then noResultsInfo after both of them

File: libs/translations/src/feedback/en.json (hunk 1 above)
Comment (aniasgit) [reply libs/translations/src/feedback/en.json:9]: done

File: libs/translations/src/common/en.json (hunk 3 above)
Comment (aniasgit) [reply libs/translations/src/common/en.json:19]: done

**Review by piotrrawski — APPROVED** [review]

File: libs/translations/src/common/en.json [hunk 5]
```diff
@@ -6,14 +6,14 @@
     "feedback-external": "Survey Participant"
   },
   "homepage": {
```
Comment (kacperChlon) [inline libs/translations/src/common/en.json:8]: please move this to feedback translations, due ti the fact presentations are related to feedback only

File: libs/translations/src/common/en.json [hunk 6]
```diff
@@ -6,14 +6,14 @@
...
-    "newPresentationDescription": "Placeholder text for ActionCard to create new presentation on Homepage",
-    "newPresentationButton": "New presentation"
+    "newPresentationButton": "New presentation",
+    "newPresentationDescription": "Placeholder text for ActionCard to create new presentation on Homepage"
```
Comment (kacperChlon) [inline libs/translations/src/common/en.json:10]: Let's add here some better translation

File: libs/translations/src/common/en.json [hunk 7]
```diff
@@ -6,14 +6,14 @@
...
   },
   "intive": {
-    "name": "Intive",
-    "wroclaw": "Wroclaw",
     "address": "Pilsudskiego 69, 50-019 Wroclaw",
-    "phone": "+48 71 769 59 00"
+    "name": "Intive",
+    "phone": "+48 71 769 59 00",
+    "wroclaw": "Wroclaw"
```
Comment (kacperChlon) [inline libs/translations/src/common/en.json:16]: WrocLaw?

**Comment by kacperChlon** [comment]
If you need to sort json object alphabetically (for insntance translations) use extensionn in vscode that hanndles such case. It will be much easier to just ctrl + shift + p -> sort json

File: libs/translations/src/common/en.json (hunk 5 above)
Comment (aniasgit) [reply libs/translations/src/common/en.json]: done

File: libs/translations/src/common/en.json (hunk 6 above)
Comment (aniasgit) [reply libs/translations/src/common/en.json]: done

**Pull request merged**